### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_ScreenReader: The identical names provided for multiple “64 bit” and “32 bit” interactive elements.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2803]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 2.4.4

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate to through the "Download Insiders" screen.
5.	Start the screen reader.
6.	Navigate to the "64 bit" and "32 bit" interactive elements in the table and listen how it is announced.

**Actual:**
When screen reader users navigate to the "64 bit" and "32 bit" interactive elements present in the table then the users will not know which element is for which content.

Similar was observed in "Edge + Narrator" as well as in "Chrome + NVDA".

**Expected:**
When screen reader users navigate to the "64 bit" and "32 bit" interactive elements of "User Installer" it should be announced as "User Installer for 64 bit" and "User Installer for 32 bit" respectively.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the multiple interactive elements have the same name then the screen reader users will have difficulty in identifying which interactive element is from which section on the screen.

**MAS Reference:**
MAS 2.4.4 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7})

**Attachment for Reference:**
[Code.visualstudio.com_DownloadInsiders_ScreenReader Identical names for multiple interactive elements..pptx](https://github.com/microsoft/vscode-docs/files/3320723/Code.visualstudio.com_DownloadInsiders_ScreenReader.Identical.names.for.multiple.interactive.elements.pptx)
